### PR TITLE
Add best practices for comparing load-tests

### PR DESCRIPTION
**Pull request by agnivade**
This PR adds some best practices that are useful when comparing results from different load-tests.

### CODE REVIEW

File: docs/compare.md [hunk 1]
```diff
@@ -22,4 +22,18 @@ To compare two load test reports, run:
...
\ No newline at end of file
+The results.txt will be a markdown formatted table comparing the average and p99 times of the store and API metrics. Additionally, a `--graph` parameter can also be passed which can be used to generate graphs comparing different metrics like CPU, Memory etc. This also requires the `gnuplot` command to be installed on the system for it to plot graphs.
+
+## Best practices while comparing load-tests
+
+- Always use the same cluster setup to compare different tests.
+- Calculate the number of users required to reach an average time of at least 10ms. And then compare the tests using that number of users. Anything less than that means the DB is not being stressed well enough and can introduce a lot of noise in the results.
+- While starting another test, it is recommended to reset the DB and initialize it again to start with the exact same state as the previous test. Follow these steps to accomplish that.
+
+On the server, run:
```
Comment (streamer45) [inline docs/compare.md:33]:
```suggestion
On the app server's instance, run:
```

File: docs/compare.md [hunk 2]
```diff
@@ -22,4 +22,18 @@ To compare two load test reports, run:
...
+- `/opt/mattermost$ echo YES | ./bin/mattermost reset`
+- `/opt/mattermost$ ./bin/mattermost user create --email sysadmin@sample.mattermost.com --username sysadmin --password Sys@dmin-sample1 --system_admin`
+- `sudo service mattermost restart`
+
+Then on the coordinator, run:
```
Comment (streamer45) [inline docs/compare.md:38]:
```suggestion
Then on the coordinator's instance, run:
```

File: docs/compare.md [hunk 3]
```diff
@@ -22,4 +22,18 @@ To compare two load test reports, run:
...
+## Best practices while comparing load-tests
+
+- Always use the same cluster setup to compare different tests.
+- Calculate the number of users required to reach an average time of at least 10ms. And then compare the tests using that number of users. Anything less than that means the DB is not being stressed well enough and can introduce a lot of noise in the results.
+- While starting another test, it is recommended to reset the DB and initialize it again to start with the exact same state as the previous test. Follow these steps to accomplish that.
+
+On the server, run:
+- `/opt/mattermost$ echo YES | ./bin/mattermost reset`
+- `/opt/mattermost$ ./bin/mattermost user create --email sysadmin@sample.mattermost.com --username sysadmin --password Sys@dmin-sample1 --system_admin`
+- `sudo service mattermost restart`
```
Comment (streamer45) [inline docs/compare.md:36]: Did we ever have a ticket to make this work through `ltctl`?

File: docs/compare.md [hunk 4]
```diff
@@ -22,4 +22,18 @@ To compare two load test reports, run:
...
+
+Then on the coordinator, run:
+- `~/mattermost-load-test-ng$ ./bin/ltagent init`
```
Comment (streamer45) [inline docs/compare.md:39]: Does it make sense to use absolute paths to avoid the user even changing directory or figuring out that's needed? :D

Comment (agnivade) [reply docs/compare.md:39]: Fine .. :expressionless:

File: docs/compare.md (hunk 3 above)
Comment (agnivade) [reply docs/compare.md:36]: No, I don't think so. Created https://mattermost.atlassian.net/browse/MM-26443.

File: docs/compare.md (hunk 4 above)
Comment (streamer45) [reply docs/compare.md]: Was this a reluctant yes? :D It's not a big deal just a detail.

Comment (agnivade) [reply docs/compare.md]: Reluctant in a funny way. It's changed now :)

**Pull request merged**